subt/docker/robotika/Dockerfile

```diff
@@ -3,11 +3,11 @@ FROM osrf/subt-virtual-testbed:subt_solution_latest
 RUN /usr/bin/python3 -m pip install --user --no-cache-dir msgpack pyserial "opencv-python>3.4.2,<4" pyzmq
 RUN git clone https://github.com/robotika/osgar.git osgar \
   && cd osgar \
-  && git checkout feature/virtual-ignition
+  && git checkout feature/virtual-tunnel-main
```

Comment:
Zbynek, do you have suggestion what branch to put here? Or are we going to create artificial `feature/virtual-urban` which we will regularly rebase on master?